dubbo-remoting/dubbo-remoting-zookeeper/src/main/java/org/apache/dubbo/remoting/zookeeper/curator/CuratorZookeeperClient.java

```diff
@@ -36,10 +42,11 @@
 import java.nio.charset.Charset;
 import java.util.Collections;
 import java.util.List;
+import java.util.concurrent.Executor;
 
-public class CuratorZookeeperClient extends AbstractZookeeperClient<CuratorWatcher> {
+public class CuratorZookeeperClient extends AbstractZookeeperClient<TreeCacheListener, CuratorZookeeperClient.CuratorWatcherImpl> {

```

Comment:
Since `CuratorWatcherImpl` implements both TreeCacheListener and CuraorWatcher, can we use CuratorWatcherImpl directly?